Mathlib/Data/Set/Ncard.lean

```diff
@@ -766,4 +776,168 @@ theorem ncard_eq_three : s.ncard = 3 ↔ ∃ x y z, x ≠ y ∧ x ≠ z ∧ y 
   tauto
 #align set.ncard_eq_three Set.ncard_eq_three
 
+section encard
+
+/-- The cardinality of a `Set` as an term in `ENat`. Infinite sets have cardinality `⊤`  -/
+noncomputable def encard (s : Set α) : ℕ∞ := PartENat.withTopEquiv (PartENat.card s)
+
+theorem Finite.encard_eq (hs : s.Finite) : s.encard = (s.ncard : ℕ∞) := by
```

Comment:
Just to clarify, the capitalization is *correct* here because it is part of the namespace and makes dot notation work properly.